middleware.ts

```diff
@@ -1,25 +1,30 @@
 import { NextResponse, NextRequest } from 'next/server';
 
 // TODO: These routes will need to be updated.
-const protectedPaths = ['/dmps/', '/admin/', '/template']
-
+const excludedPaths = ['/login', '/signup', '/email', '/favicon.ico', '/_next', '/api'];
```

Comment:
Switched to using excludedPaths instead of protected paths, since most pages will be protected.